### ISSUE TITLE
Copy/paste IP from Admin UI should not contain spaces

### ISSUE THREAD

**Issue by drasch** [tailscale/tailscale#1850]
**Is your feature request related to a problem? Please describe.**

When visiting the admin UI and trying to copy the IP address for a node (in Firefox), I triple-click on the IP address to select this text. When I go to paste this in a terminal, it contains preceding spaces. For example: <pre>    127.0.0.127</pre>.

**Describe the solution you'd like**

The markup HTML changed or adding a click-to-copy feature.

**Describe alternatives you've considered**

For now, I select the IP manually with a click-drag.

**Comment by rosszurowski** [comment]
Which page and OS is this on? When I copy an IP address from Firefox on macOS it doesn't add preceding spaces.

We do plan to introduce a click-to-copy feature soon — sorry for these troubles in the meantime.

**Comment by drasch** [comment]
Originally I ran into this bug on Ubuntu 20.04 with the steps: 
1. triple-click on the IP address on the https://login.tailscale.com/admin/machines page
2. `cmd-c` (or `ctrl-c` on Linux)
3. navigate to terminal window
4. `cmd-v`

Actual result (IP has been changed): 
<pre>    127.0.0.127</pre>

Expected result:
<pre>127.0.0.127</pre>


I just reproduced on Firefox 84.0.2 and 88.0.1 on Mac running 10.15.7 and on Linux running Firefox 88.0.

Note 1: In Firefox a double-click will select one octet of the address.

Note 2: In Chrome, a double-click will select the entire IP address.

Note 3: In Chrome, a triple-click will select the IP address AND a newline.
